spec/requests/cookies_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Cookies API", type: :request do
+  describe "PUT /cookies" do
```

Comment:
Could you also have a test case for text/html

Reply:
Per the meeting we just had, isn't that covered by the Cypress tests now?

(I'm still a bit unsure where the line is)

Reply:
I think cypress tests do indeed cover it. They also cover the case for json, right? 

Anyway, if we want to keep this test, maybe we want to name it `PUT /cookies with json` to make it obvious what it covers.

Reply:
The cypress tests don't test for `status: ok` or anything else that we add to the response in the future - but I guess that's just because it's not actually being used